cms/templates/js/course-video-settings-update-settings-footer.underscore

```diff
@@ -0,0 +1,13 @@
+<button class='button-link action-cancel-course-video-settings' aria-describedby='cancel-button-text'>
+    <%- gettext('Reset') %>
+    <span id='updcancel-button-text' class='sr'><%-gettext('Click Reset to discard changes.') %></span>
```

Comment:
Same comment for these two lines as for course-video-settings-update-org-credentials-footer.underscore

Reply:
Actually this is screen reader text, so it won't be shown.